code/game/machinery/vending/vendor_types/medical.dm

```diff
@@ -82,6 +123,10 @@
 			to_chat(user, SPAN_WARNING("[src] makes a warning noise. The [C.name] is currently full."))
 			return
 
+		if(requires_supply_link_port && !get_supply_link())
+			to_chat(user, SPAN_WARNING("\The [src] needs to be linked to a medical supply link port to refill the [C.name]!"))
```

Comment:
This will confuse players, they'll think they need to find out what a medical supply link is  and go 'hey does anyone know where i can find a supply link?'

```suggestion
			to_chat(user, SPAN_WARNING("\The [src] makes a buzzing noise as it rejects \the [C.name]. Looks like this vendor cannot refill these outside of a medical bay's supply link."))
			playsound(src, 'sound/machines/buzz-sigh.ogg', 15, TRUE)
```